### PR TITLE
Refs #29539 - Unbundle websockify

### PR DESCRIPTION
**Pull request by ekohl**
This reflects the removal of a bundled websockify in favor of a system package. While it is in EPEL8 (and the spec is copied from that), Foreman doesn't support running with EPEL8 enabled.

Draft until the core PR is merged: https://github.com/theforeman/foreman/pull/9392

### CODE REVIEW

**Comment by ekohl** [comment]
Now fails on:
```
*** ERROR: ambiguous python shebang in /usr/share/foreman/extras/noVNC/websockify.py: #!/usr/bin/python. Change it to python3 (or python2) explicitly.
*** ERROR: ambiguous python shebang in /usr/share/foreman/extras/noVNC/websockify/websocketproxy.py: #!/usr/bin/env python. Change it to python3 (or python2) explicitly.
```

After core has been merged those files should be gone.

File: packages/foreman/python-websockify/python-websockify.spec
Comment (evgeni) [inline packages/foreman/python-websockify/python-websockify.spec:9]: Technically, the correct home is at https://github.com/novnc/websockify

**Review by evgeni — APPROVED** [review]

File: packages/foreman/python-websockify/python-websockify.spec
Comment (ekohl) [reply packages/foreman/python-websockify/python-websockify.spec:9]: I swear I fixed that, but then I regenerated it and copied it from EPEL. I'll submit a PR to Fedora to fix it.

**Comment by ekohl** [comment]
I cancelled the build: it's waiting for https://ci.theforeman.org/job/foreman-develop-source-release/1733/ to finish.

**Comment by ekohl** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Review by evgeni — APPROVED** [review]

**Pull request merged**
